mariadb_skysql/hatch.toml

```
@@ -0,0 +1,4 @@
+[env.collectors.datadog-checks]
+
+[[envs.default.matrix]]
+python = ["2.7", "3.8"]
```

Assuming you don't intend to support Python 2 (which is most likely):

```suggestion
python = ["3.8"]
```

That will fix the current test failure.